### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Arthusppp**
Update Polish translation

### CODE REVIEW

File: files/lang/pl.po [hunk 1]
```diff
@@ -5193,38 +5186,32 @@ msgstr "Ustaw armię w luźnej formacji"
...
+msgstr "Pokaż poprzedniego bohatera"
 
-#, fuzzy
 msgid "Show next hero"
 msgstr "Pokaż następnego bohatera"
 
-#, fuzzy
 msgid "Blood Morale"
-msgstr "Wysokie morale"
+msgstr "Krwi!"
```
Comment (ihhub) [inline files/lang/pl.po:5208]: I think this message is used when we show a popup window for moral and it includes in the sentence. If it's true then we shouldn't set an exclamation mark at the end.

File: files/lang/pl.po [hunk 2]
```diff
@@ -5294,26 +5281,21 @@ msgstr ""
 "Twój bohater jest zbyt zmęczony, by rzucić dziś to zaklęcie. Spróbuj "
 "ponownie jutro."
 
-#, fuzzy
 msgid "%{spell} failed!!!"
 msgstr "Czar nie udany!"
```
Comment (ihhub) [inline files/lang/pl.po:5285]: It's better to keep `%{spell}` as it shows the name of the spell.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Arthusppp , I left 2 comments with my limited knowledge of Polish. Could you please take a look?

File: files/lang/pl.po (hunk 1 above)
Comment (Arthusppp) [reply files/lang/pl.po:5208]: This is somehow easter egg from the OG which I like very much!

image

File: files/lang/pl.po (hunk 2 above)
Comment (Arthusppp) [reply files/lang/pl.po:5285]: Oh gosh, you are totally right! I am updating it.

File: files/lang/pl.po (hunk 1 above)
Comment (Arthusppp) [reply files/lang/pl.po:5208]: The fheroes2 screen: 
image

Comment (ihhub) [reply files/lang/pl.po:5208]: This window uses the following translation:
```
msgid "morale|Blood!"
msgstr "Krwi!"
```
which is present in the file. The line you've modified corresponds to a popup window if you click on the moral icon in the top right corner of hero's dialog.

Comment (ihhub) [reply files/lang/pl.po:5208]: image

Comment (Arthusppp) [reply files/lang/pl.po:5208]: Oh gosh, I have missed it. I will withdraw my change.

File: files/lang/pl.po (hunk 2 above)
Comment (michkry) [reply files/lang/pl.po]: I think we should change "nie udany" to "nieudany", in addition to the change exposed by ihhub

File: files/lang/pl.po [hunk 3]
```diff
@@ -7369,44 +7340,38 @@ msgstr ""
 msgid "Endless Pouch of Sulfur"
 msgstr "Bezdenna sakiewka siarki"
 
-#, fuzzy
 msgid "The %{name} provides %{count} unit of sulfur per day."
-msgstr "%{name} zapewnia %{count} jednostkę siarki na dzień"
+msgstr "%{name} daje ci %{count} jednostkę siarki dziennie."
```
Comment (michkry) [inline files/lang/pl.po:7344]: In my opinion "zapewnia" fits better than "daje ci". It reflects more accurately the meaning of the original sentence

File: files/lang/pl.po [hunk 4]
```diff
@@ -7369,44 +7340,38 @@ msgstr ""
...
 
 msgid "Endless Vial of Mercury"
 msgstr "Bezdenna fiolka rtęci"
 
-#, fuzzy
 msgid "The %{name} provides %{count} unit of mercury per day."
-msgstr "%{name} zapewnia %{count} jednostkę rtęci dziennie"
+msgstr "%{name} daje ci %{count} jednostkę rtęci dziennie."
```
Comment (michkry) [inline files/lang/pl.po:7350]: Same as above

File: files/lang/pl.po [hunk 5]
```diff
@@ -7369,44 +7340,38 @@ msgstr ""
...
 
 msgid "Endless Pouch of Gems"
 msgstr "Bezdenna sakiewka klejnotów"
 
-#, fuzzy
 msgid "The %{name} provides %{count} unit of gems per day."
-msgstr "%{name} zapewnia %{count} klejnot dziennie"
+msgstr "%{name} daje ci %{count} jednostkę klejnotów dziennie."
```
Comment (michkry) [inline files/lang/pl.po:7356]: Same as above

File: files/lang/pl.po [hunk 6]
```diff
@@ -7724,50 +7688,46 @@ msgstr ""
...
 msgid ""
 "Do you want to use your knowledge of magical secrets to transcribe the "
 "%{spell} Scroll into your Magic Book?\n"
 "The Spell Scroll will be consumed.\n"
 " Cost in spell points: %{sp}"
 msgstr ""
-"Czy chcesz wykorzystać swoją wiedzę magiczną aby zapisać %{spell} zwój do "
-"swojej księgi czarów ?"
+"Czy chcesz wykorzystać swoją wiedzę magiczną aby zapisać %{spell} ze zwoju "
+"do swojej księgi czarów?\n"
```
Comment (michkry) [inline files/lang/pl.po:7698]: In the line 5149 we refer to Spell Book as "Księga Magii". Here "Magic Book" is called "Księga czarów". Shouldn't we use "Księga Magii" here as well?

File: files/lang/pl.po [hunk 7]
```diff
@@ -7724,50 +7688,46 @@ msgstr ""
...
+"Zwój zostanie wykorzystany.\n"
+"Koszt punktów magii: %{sp}"
 
-#, fuzzy
 msgid "Move points: %{mp}"
-msgstr "Ptk. ruchu"
+msgstr "Koszt punktów ruchu: %{mp}"
```
Comment (michkry) [inline files/lang/pl.po:7703]: I'm unaware of the context in which this literal is used but I think that "Punkty ruchu" expresses better its original meaning

File: files/lang/pl.po [hunk 8]
```diff
@@ -8390,7 +8349,7 @@ msgid ""
 msgstr ""
 
 msgid "heroes: allow buy a spellbook from Shrines"
-msgstr ""
+msgstr "bohaterowie: pozwól kupować księgę magii z kaplic"
```
Comment (michkry) [inline files/lang/pl.po:8352]: I think that we should change "z kaplic" to "w kaplicach"

File: files/lang/pl.po [hunk 9]
```diff
@@ -7340,8 +7313,7 @@ msgstr "Pierścień mocy"
 #, fuzzy
 msgid "The %{name} returns %{count} extra spell points per day to your hero."
 msgstr ""
-"%{name} przywraca %{count} dodatkowych punktów mocy na turę dla twojego "
-"bohatera"
+"%{name} przywraca twemu bohaterowi dodatkowe %{count} punkty magii na turę."
```
Comment (michkry) [inline files/lang/pl.po:7316]: Shouldn't we use "twojemu" instead of "twemu"?

File: files/lang/pl.po [hunk 10]
```diff
@@ -7544,7 +7509,6 @@ msgstr "Szpadel Nekromancji"
 msgid "The %{name} gives you increased necromancy skill."
 msgstr "%{name} zwiększa twoją umiejętność nekromancji."
```
Comment (michkry) [inline files/lang/pl.po:7510]: I think we should change "twoją umiejętność" to "twoje umiejętności"

File: files/lang/pl.po (hunk 3 above)
Comment (Arthusppp) [reply files/lang/pl.po:7344]: Ok, let's change all towards "zapewnia".

File: files/lang/pl.po (hunk 6 above)
Comment (Arthusppp) [reply files/lang/pl.po:7698]: Correct and correct! I am updating the translation! :)

//and I promise, some day I will look holistically at all inconsistencies like this: "księga magii/księga zaklęć", "stwór/potwór/jednostka", "czar/zaklęcie", etc. ;)

File: files/lang/pl.po (hunk 7 above)
Comment (Arthusppp) [reply files/lang/pl.po:7703]: I suggest to keep as I proposed. The whole message here is:

> "Czy chcesz wykorzystać swoją wiedzę magiczną, aby zapisać %{spell} ze zwoju "
> "do swojej księgi zaklęć?\n"
> "Zwój zostanie wykorzystany.\n"
> "Koszt punktów magii: %{sp}"
> "Koszt punktów ruchu: %{mp}"

File: files/lang/pl.po (hunk 8 above)
Comment (Arthusppp) [reply files/lang/pl.po:8352]: Ok.

File: files/lang/pl.po (hunk 9 above)
Comment (Arthusppp) [reply files/lang/pl.po:7316]: "Krótsze formy (np. ma, me, mej, twe) mają charakter książkowy i są używane rzadko." So - fheroes2 is not a book! Changing to "twojemu"! Thanks for catching this!

File: files/lang/pl.po (hunk 10 above)
Comment (Arthusppp) [reply files/lang/pl.po:7510]: I do not think so, as the "nekromancja" is singular only.

**Comment by ihhub** [comment]
@Arthusppp , thank you so much for these changes!

**Pull request merged**